server/src/main/java/com/epam/aidial/core/server/Proxy.java

```diff
@@ -280,6 +281,11 @@ private void respond(HttpServerRequest request, HttpStatus status) {
     }
 
     private void respond(HttpServerRequest request, HttpStatus status, String body) {
+        if (status == HttpStatus.OK) {
+            SpanUtil.updateName(request.method().name(), request.path());
```

Comment:
Why do we update span name again? Moreover we set a raw path.

So we do have different span names based on http codes. Why?

Reply:
Will be fixed in scope of a separate issue